automation_tools/__init__.py

```diff
@@ -1000,11 +1000,13 @@ def product_install(distribution, create_vm=False, certificate_url=None,
 
     if distribution.startswith('satellite6'):
         execute(setup_default_docker, host=host)
-        execute(setup_oscap, host=host)
         execute(katello_service, 'restart', host=host)
         execute(install_puppet_scap_client, host=host)
         execute(setup_foreman_discovery, host=host)
 
+    if not distribution.endswith('upstream'):
```

Comment:
Please indent this if block to be inside the satellite distribution as we use this for sam too.